Bottom: 20cf969fc852aae4ea6dec5c4d5b50dce09f099b
Top:    fe7cbbcfd2f8b09c7a76e754eac0d795eb312691
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-10-14 13:11:55 +0100

Refresh of add-debug-catch-to-triangle-ha

---

diff --git a/src/hid/common/hidgl.h b/src/hid/common/hidgl.h
index 1980195..315235c 100644
--- a/src/hid/common/hidgl.h
+++ b/src/hid/common/hidgl.h
@@ -47,7 +47,7 @@ hidgl_add_triangle (triangle_buffer *buffer,
                     GLfloat x3, GLfloat y3)
 {
   if (buffer->triangle_count == TRIANGLE_ARRAY_SIZE) {
-    printf ("OH CRAP\n");
+    fprintf (stderr, "hidgl_add_triangle: Attempt to add past end of array. Will now segfault.\n");
     *(char *)0 = 0;
   }
   buffer->triangle_array [buffer->coord_comp_count++] = x1;
